extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -62,8 +63,7 @@ const EXPANDABLE_COMPONENTS = {
     '2 102.92 0 0 1-63.7 22 104.41 104.41 0 0 1-12.21-.74 145.21 145.21 0 0 ' +
     '0 78.62 23"/></g></svg>',
     actionIcon: ActionIcon.EXPAND,
-    // TODO(enriqe): change for i18n label.
-    defaultText: 'View on Twitter',
+    defaultText: LocalizedStringId.AMP_STORY_TOOLTIP_EXPAND_TWEET,
```

Comment:
I'm sorry but I still feel like this should be renamed, it says `defaultText`, so we'd expect a string, when it's actually a `LocalizedStringId` :(